src/hb-ot-var-common.hh

```diff
@@ -1953,9 +1951,6 @@ struct item_variations_t
       encodings.push (std::move (encoding_objs.arrayZ[i]));
     }
 
-    /* sort again based on width, make result deterministic */
```

Comment:
iirc, this sorting seems required for a deterministic result during my testing(I don't remember why we'd have non-deterministic behavior) and we might need to keep it to match fonttools' result. Fonttools also does sorting twice: see: [1st,](https://github.com/fonttools/fonttools/blob/03a3c8ed9e7ab35b5f219a6d25ee8081107ff2bb/Lib/fontTools/varLib/varStore.py#L601) [2nd](https://github.com/fonttools/fonttools/blob/03a3c8ed9e7ab35b5f219a6d25ee8081107ff2bb/Lib/fontTools/varLib/varStore.py#L649)